pkg/internal/config/config.go

```diff
@@ -70,16 +70,18 @@ const (
 )
 
 var (
-	ErrInvalidNetworkAttachmentDefinitionName   = errors.New("invalid Network-Attachment-Definition Name")
-	ErrInvalidTrafficGeneratorRuntimeClassName  = errors.New("invalid Traffic Generator Runtime class Name")
-	ErrInvalidTrafficGeneratorPacketsPerSecond  = errors.New("invalid Traffic Generator Packets Per Second")
-	ErrInvalidPortBandwidthGB                   = errors.New("invalid Port Bandwidth [GB]")
-	ErrInvalidTrafficGeneratorEastMacAddress    = errors.New("invalid Traffic Generator East MAC Address")
-	ErrInvalidTrafficGeneratorWestMacAddress    = errors.New("invalid Traffic Generator West MAC Address")
-	ErrInvalidDPDKEastMacAddress                = errors.New("invalid DPDK East MAC Address")
-	ErrInvalidDPDKWestMacAddress                = errors.New("invalid DPDK West MAC Address")
-	ErrInvalidTestDuration                      = errors.New("invalid Test Duration")
-	ErrInvalidVerbose                           = errors.New("invalid Verbose value [true|false]")
+	ErrInvalidNetworkAttachmentDefinitionName  = errors.New("invalid Network-Attachment-Definition Name")
+	ErrInvalidTrafficGeneratorRuntimeClassName = errors.New("invalid Traffic Generator Runtime class Name")
+	ErrIllegalLabelSelectorCombination         = errors.New("illegal Traffic Generator and DPDK Node " +
+		"Label Selector combination")
```

Comment:
nit: Is it possible to have the error message on a single line?

Reply:
nope. lint..